docs/kubernetes/README.md

```diff
@@ -143,6 +143,15 @@ When using XFS as the filesystem type and no `fsFormatOptions` are set, we apply
 
 If you set any options at all, it is your responsible to make sure that all default flags from `mkfs.xfs` are supported on your current Linux Kernel version or that you set the flags appropriately.
 
+### Volume Location
```

Comment:
https://github.com/hetznercloud/csi-driver/blob/63cd777f24b215102c2f45ce1c880baa58bf763b/internal/driver/controller.go#L61-L75

I could only find this snippet where we evaluate the accessibility requirements and set the location, otherwise defaulting. Could you maybe point me to the right lines of code?

Reply:
Otherwise, I would add an additional sentence to the docs.

Reply:
That the right snippet actually. Looks like the csi-provisioner already takes the location label from the Node and puts it into the AccessibilityRequirements.